mne_bids/config.py

```diff
@@ -147,12 +149,17 @@
     ".snirf",  # SNIRF
 ]
 
+allowed_extensions_motion = [
+    ".tsv",  # Tab-separated values
+]
+
```

Comment:
This seems to be the line causing the issue, which makes sense because the EEG BIDS datasets have sidecar files that are just text files. So adding `.tsv` to the allowed datafile extensions seems to be confusing some of mne_bids internal functions..

Motion data might be the first case where data are stored in simple text files, which is why we are hitting this problem now. But I know that Eye-tracking BIDS will have the same issue..  So supporting these data might require more refactoring of the codebase.

Reply:
As a grumpy sidenote, I never understood why BIDS encourages storing some modalities of data in text files... !

Reply:
without having looked into it too deeply, that could be something that needs refactoring, yes.